shap/explainers/_deep/__init__.py

```diff
@@ -87,14 +85,16 @@ def __init__(self, model, data, session=None, learning_phase_flags=None):
         super().__init__(model, masker)
 
         if framework == 'tensorflow':
+            from .deep_tf import TFDeep
             self.explainer = TFDeep(model, data, session, learning_phase_flags)
         elif framework == 'pytorch':
+            from .deep_pytorch import PyTorchDeep
```

Comment:
Out of interest, what is the motivation behind moving these imports?

Reply:
mypy complaints since previously we created empty global variables in the tensorflow repository (tf, tf_backprop, etc.). When I removed those and imported tf at the top of the deepTf file I had to move these imports down here, so that the pytorch DeepExplainer didn't need a tensorflow dependency

Reply:
Ok, that makes sense. In fact I think this approach of importing the optional dependencies in-line like this is simpler and consistent with #3533